app/models/translation_override.rb

```diff
@@ -44,6 +44,7 @@ class TranslationOverride < ActiveRecord::Base
   validates_presence_of :locale, :translation_key, :value
 
   validate :check_interpolation_keys
+  validate :check_MF_string, if: :message_format?
```

Comment:
What if there is an existing bad override in the database? In that case, I think this validation will prevent us from updating it, even if we're just changing the 'status' to deprecated/outdated?

I wonder if the validation should go in the controller instead, so we're only running it in direct response to user input?

Reply:
I’ve combed through the code and the only place where this could have been problematic was in `SiteTextsController`. I’ve extracted the logic of making an override up-to-date directly in the model as it makes more sense. This bypasses the validations so everything should still work as before :slightly_smiling_face: 

This way, we can still inspect any override very easily, checking which ones are invalid. Also, the change is minimal, I didn’t need to touch the controller much more than that.